src/components/SearchableDropdownListItem.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const SearchableDropdownListItem = ({ place, handleListItemClick }) => (
```

Comment:
Looking at this code could you really tell that this is something Searchable and from DropdownList ?